shards/modules/egui/src/widgets/code_editor/sublime-syntax.yml

```diff
@@ -44,7 +44,10 @@ contexts:
     - match: '[a-z_][a-zA-Z0-9_.-]*'
       scope: variable.other.readwrite.shards
   strings:
-    - match: '".*"'
-      scope: string.quoted.double.shards
-    - match: '""".*"""'
-      scope: string.quoted.triple.shards
+    - match: '"|"""'
+      scope: string.quoted.shards
+      push:
+        - meta_content_scope: string.quoted.shards
+        - match: '"|"""'
```

Comment:
why the `|`?

Reply:
To match a double quote or a triple double quote.
I'm going to make it a bit more correct.

Reply:
Oh so unrelated to our `|`?

Reply:
Yup, unrelated, it's just an "or" for a regex